modular_citadel/code/modules/mob/living/carbon/human/species_types/species.dm

```diff
@@ -0,0 +1,12 @@
+//shamelessly stolen from hippie
+/obj/item/bodypart/var/should_draw_hippie = FALSE

```

Comment:
You can replace all references to hippie with cit.

Also, would ya be aight with porting over all of cit's species to use the should_draw_citadel vars/procs?

Reply:
i was gonna change it to citadel instead of draw_hippie, but i kept it cause meme